common/source/docs/common-accelerometer-calibration.rst

```diff
@@ -44,9 +44,11 @@ Calibration steps
       Accelerometer Calibration Positions (Copter)
 
    It is important that the vehicle is kept still immediately after pressing the key for each step. This is more important than getting the angle exactly right, ie. left being 90deg to horizontal, etc. Except for the first "LEVEL", the positions can be within 20 degs of being exact. Being still in each position as you press the key is much more important.
-
+   
+   -  You should calibrate the board mounted in the vehicle if possible. 
    -  You may need to calibrate the board before it is mounted if the size/shape of the vehicle makes this difficult.
    -  The level position is the most important to get right as this will be the attitude that your controller considers level while flying.
+   -  You can recalibrate LEVEL using Mission Planner, after you have installed the flight controller and are ready to fly.
```

Comment:
Drop Mission Planner and use "AHRS trim" instead.

Reply:
Is this documentation intended for someone new to this? If I am new to this, this documentation is already pretty hard to follow. Dropping back to "AHRS Trim" might not help making this more accessible to folks trying to get started.

Reply:
AHRS_TRIM is not a param that we recommend for a user to access....we used to recommend using it to adjust AOA after a bench or fuselage line accel cal, but it interacts with VTOL setup and TRIM_PITCH_CD is the only param we suggest a user touch....unfortunately, copter and heli dont have this param and must use it
I really need to scrub it from the two plane docs that still have it...

Reply:
I think I will leave the refs in tuning-cruise since it shows using AHRS trims as being NOT the preferred method already...and just change the starting up and calibrating doc...

Reply:
#4131